AcceptanceTests/runAcceptanceTests.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/sh
 echo "📴📴📴📴📴📴📴 Acceptance Testing has started 📴📴📴📴📴📴📴"
 
+BUILDDIR=$(xcodebuild -configuration Debug -showBuildSettings | grep "CONFIGURATION_BUILD_DIR" | grep -oEi "\/.*")
```

Comment:
I'd love to see a comment about the grep part. This is a good starting point: https://explainshell.com/explain?cmd=grep+-oEi+%22%5C%2F.*%22